container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -8,12 +8,13 @@ const (
 	rootDirpath = "/fluent-bit"
 
 	////////////////////////--FLUENT BIT CONTAINER CONFIGURATION SECTION--/////////////////////////////
-	containerImage        = "fluent/fluent-bit:1.9.7"
+	containerImage        = "fluent/fluent-bit:4.0.0"
```

Comment:
This is a big jump; just verifying that there aren't any problems with doing this upgrade?

Reply:
afaict, everything,s been good so far, config from 1.9.7 still works and all the config still takes into effect